new_platforms/vscode-extension/assets/baseSolutionTemplateFolder/enc/enc.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/bits/stdio.h>
+
+// Include the trusted helloworld header that is generated
+// during the build. This file is generated by calling the
+// sdk tool sgx_edger8r against the helloworld.edl file.
```

Comment:
No it isn't.   We use oe_edger8r not sgx_edger8r.  This is especially important since you don't even support SGX at all with this extension today.